<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>336</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>336</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>336</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PortableApps.com Development Toolkit</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../resources/graphics.qrc">
    <normaloff>:/icon.png</normaloff>:/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="sidebarLogo">
      <property name="text">
       <string notr="true"/>
      </property>
      <property name="pixmap">
       <pixmap resource="../../../resources/graphics.qrc">:/sidebar.png</pixmap>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="pageLayout">
      <property name="margin">
       <number>9</number>
      </property>
      <item>
       <widget class="QCommandLinkButton" name="createButton">
        <property name="statusTip">
         <string>Start a new portable app package</string>
        </property>
        <property name="text">
         <string>&amp;Create a new portable app...</string>
        </property>
        <property name="description">
         <string>Start a new portable app package here</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="packageLayout">
        <item>
         <widget class="QLabel" name="packageLabel">
          <property name="text">
           <string>&amp;Package:</string>
          </property>
          <property name="buddy">
           <cstring>packageText</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="packageText"/>
        </item>
        <item>
         <widget class="QPushButton" name="packageButton">
          <property name="text">
           <string>&amp;Browse...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="detailsButton">
        <property name="statusTip">
         <string>Edit the basic app details</string>
        </property>
        <property name="text">
         <string>Edit &amp;details</string>
        </property>
        <property name="description">
         <string>Edit the basic app details</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="validateButton">
        <property name="statusTip">
         <string>Validate PortableApps.com Format compliance</string>
        </property>
        <property name="text">
         <string>&amp;Validate</string>
        </property>
        <property name="description">
         <string>Validate the app</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="installerButton">
        <property name="statusTip">
         <string>Generate your installer</string>
        </property>
        <property name="text">
         <string>PortableApps.com &amp;Installer</string>
        </property>
        <property name="description">
         <string>Generate your installer</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../../resources/graphics.qrc"/>
 </resources>
 <connections/>
</ui>
